src/models/StuntSheet.ts

```diff
@@ -8,7 +8,7 @@ import { loadContinuity } from './continuity/load-continuity';
  * Defines the positions/directions in a formation and the continuities
  * used to reach the next position.
  *
- * @property title          - High level description
+ * @property title          - Title of this StuntSheet
```

Comment:
For property descriptions, it's more helpful if you use words that are not the property name. In this case, using `Title of this Stuntsheet` to describe `title` doesn't really add any additional insight into what `title` is used for... After all, the property name is `title` inside the `Stuntsheet` model.